pkg/transform/transform.go

```diff
@@ -0,0 +1,158 @@
+package transform
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/fusor/cpma/env"
+	"github.com/fusor/cpma/pkg/io"
+	"github.com/fusor/cpma/pkg/transform/oauth"
+	"github.com/fusor/cpma/pkg/transform/secrets"
+	"github.com/sirupsen/logrus"
+)
+
+// OCP4InstallMsg message about using generated manifests
+const OCP4InstallMsg = `To install OCP4 run the installer as follow in order to add CRDs:
+' /openshift-install --dir $INSTALL_DIR create install-config'
+'./openshift-install --dir $INSTALL_DIR create manifests'
+# Copy generated CRD manifest files  to '$INSTALL_DIR/openshift/'
+# Edit them if needed, then run installation:
+'./openshift-install --dir $INSTALL_DIR  create cluster'`
+const MasterConfigFile = "/etc/origin/master/master-config.yaml"
```

Comment:
Do we have any reason to believe these ocp3 file locations have ever changed? What if they changed between `3.x` versions? Normally I would say these should be passed by config values, but if they have never changed I think we can safely hardcode consts.

Reply:
I'm not aware of location differences with `openshift-ansible`.  We have seen differences between `openshift-ansible` and `oc cluster up` for file paths.  

To be safe, I think there is value in allowing all of these locations to be specified in config file and overridden.  

As @eriknelson mentioned with blockers and non-blockers, I would consider this a non-blocker and something we could improve in a follow up issue.

Reply:
[ISSUE]

Reply:
Just worth noting, the current implementation allows you to set the location of the master-config: https://github.com/fusor/cpma/blob/master/pkg/migration/migration.go#L77